src/src/PluginStructs/P036_data_struct.cpp

```diff
@@ -1127,7 +1167,14 @@ void P036_data_struct::P036_DisplayPage(struct EventStruct *event)
       {
         String tmpString(DisplayLinesV1[(ScrollingPages.linesPerFrame * frameCounter) + i].Content);
         ScrollingPages.LineIn[i] = P36_parseTemplate(tmpString, 20);
-
+        iAlignment=0;
+        iAlignment=get3BitFromUL(DisplayLinesV1[(ScrollingPages.linesPerFrame * frameCounter) + i].ModifyLayout, P036_FLAG_ModifyLayout_Alignment);
+        switch (static_cast<eAlignment>(iAlignment)) {
+          case eAlignment::eGlobal: ScrollingPages.TxtAlignmentIn[i]=textAlignment; break;
+          case eAlignment::eLeft:   ScrollingPages.TxtAlignmentIn[i]=TEXT_ALIGN_LEFT; break;
+          case eAlignment::eCenter: ScrollingPages.TxtAlignmentIn[i]=TEXT_ALIGN_CENTER; break;
```

Comment:
Duplicated code (lines 1131..1136), maybe move it to a separate method?

Reply:
Done!